CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 - Whether a surface is toplevel equivalent can be determined with the new function `shell::is_toplevel_equivalent`.
 - Setting the parent of a toplevel surface is now possible with the `xdg::ToplevelSurface::set_parent` function.
 - Add support for the zxdg-foreign-v2 protocol.
+- Support for `xdg_wm_base` protocol version 3
+- **[Breaking]** `XdgPositionerState` moved to `XdgPopupState` and added to `XdgRequest::NewPopup`
+- **[Breaking]** `PopupSurface::send_configure` now checks the protocol version and returns an `Result`
```

Comment:
Please put those in their own `### Breaking Changes` section, before the `### Additions` one, and maybe the two latter as sub-items of the first one, some something like that:

```
### Breaking Changes

#### Clients & Protocols

- Support for `xdg_wm_base` protocol version 3
  - `XdgPositionerState` moved to `XdgPopupState` and added to `XdgRequest::NewPopup`
  - `PopupSurface::send_configure` now checks the protocol version and returns an `Result`
```

Reply:
I moved the two breaking changes to a `### Breaking Changes` section. I left the support for protocol version 3 in the `Additions`. Is this ok for you?